app/schema_validation/__init__.py

```diff
@@ -65,39 +65,60 @@ def validate(
     json_to_validate,
     schema,
 ):
```

Comment:
nit: since you added a docstring, type annotations on the method would be a bonus, but this is a nit so feel free to ignore.